src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.split(';');
```

Comment:
The code splits the string by semicolons, which is fine, but it may include empty entries if there are consecutive semicolons. It would be more robust to filter out empty lines to avoid unnecessary iterations.